api/python/stat_value.md

```diff
@@ -60,7 +60,7 @@ In next example, the parameter "observation=P3Y" overly constrains the request s
 throws ValueError:
 
 ```python
->>> get_stat_series('geoId/06085', 'Count_Person', observation_period='P3Y')
+>>> get_stat_value('geoId/06085', 'Count_Person', observation_period='P3Y')
```

Comment:
should we also use datacommons.get_stat_value here?

Reply:
right, needs the import statement in these examples. I will correct them in next PR